docker/fundamentals/containers/docker-container-philosophy.md

```diff
@@ -14,11 +14,11 @@ category: must-know
 
 ## Content
 
-A great analogy for Docker containers can be found in the shipping industry. Before shipping containers were standardized, it was difficult to transport things across the world. With standardized containers, you just lend or buy a container, pack it with your things (in a way which makes sure the container is not negatively affected, like things sliding around) and from then on people know how to handle the container and therefore don’t need to adjust its contents too much.
+A great analogy for Docker containers can be found in the shipping industry. Before shipping containers were standardized, it was difficult to transport things across the world. Now, with advent of standardized containers, you only have to rent or buy a container and pack it with your things (in a way which makes sure not to damage the container). As the containers are standardised, people know how to handle them and therefore don’t need to adjust their contents too much.
```

Comment:
Standardized shipping containers weren't invented now so we might want to rephrase a bit more.

Maybe something like

```suggestion
A great analogy for Docker containers can be found in the shipping industry. Before shipping containers were standardized, it was difficult to transport things across the world. With the advent of standardized containers, the process became simpler and safer. Now you only have to rent or buy a container and pack it with whatever you need to transport (in a way which makes sure not to damage the container) and the rest of the process is essentially automated. With standardization, came automation, and with automation, came efficiency. 
```

Reply:
Looks good. I would make a new sentence after 'not to damage the container' and remove the 'and' so the next sentence is 'The rest of the process...' . Otherwise it's quite a long sentence.

Reply:
With regards to the last phrase, how does this sound? (changed the placing of commas)

> With standardization came automation, and with automation came efficiency.

Reply:
How about we remove the sentence in parenthesis and add a "safely" instead of it?

> Now you only have to rent or buy a container and pack it safely with whatever you need to transport and the rest of the process is essentially automated. With standardization came automation, and with automation came efficiency.